ui/projects/DeployProject.qml

```diff
@@ -0,0 +1,54 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+
+import "../../backend/core"
+import "../../backend/sccs" as Backend
+
+Item {
+    id: root
+
+    property var projectIndex
```

Comment:
can be an alias to data.projectIndex as this is not set anywhere else and only data rely on the projectIndex